scripts/superdesk-authoring/authoring.js

```diff
@@ -2352,6 +2352,7 @@
                         })
                         .then(function(value) {
                             notify.success(gettext('Item sent.'));
+                            $rootScope.$broadcast('item:fetch');
```

Comment:
Fetch is an operation to add on archive an item from ingest list. The operation needs more changes compared with a simple move.